mesa/batchrunner.py

```diff
@@ -45,7 +45,7 @@ class BatchRunner:
     entire DataCollector object.
 
     """
-    def __init__(self, model_cls, variable_parameters=None,
+    def __init__(self, model_cls, variable_parameters={},
```

Comment:
It's generally not a good practice to use a [mutable value (e.g. `{}`) as a default argument](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html). In this case I don't _think_ anything harmful would happen, but I'd suggest leaving `None` as the default value just in case, and for general Pythonic-ness.